src/main/java/org/mockito/plugins/MockitoPlugins.java

```diff
@@ -41,4 +41,19 @@ public interface MockitoPlugins {
      * @since 2.10.0
      */
     MockMaker getInlineMockMaker();
```

Comment:
Should we mark this one deprecated in favor of this new method?

Reply:
I am not sure, because the semantics are different. Maybe someone is relying on the fact, that you get a new instance every time you call it.

Reply:
I think the new behaviour is the one I would expect, not the previous one. So I would be okay with marking the unintuitive behaviour as deprecated for that reason. If somebody objects, we can always reconsider

Reply:
Done.